docs/learning-powershell/README.md

```diff
@@ -1,12 +1,12 @@
 Learning PowerShell
 ====
 
-Whether you're a Developer, a DevOps or an IT Professional, this doc will help you getting started with PowerShell.
+Whether you're a developer, a DevOps or an IT professional, this document will help you getting started with PowerShell.
 In this document we'll cover the following:
-installing PowerShell, samples walkthrough, PowerShell editor, debugger, testing tools and a map book for experienced bash users to get started with PowerShell faster.
+Installing PowerShell, samples walkthrough, PowerShell editor, debugger, testing tools, and a map book for experienced Bash users to get started with PowerShell faster.
```

Comment:
I don't think "Installing" should be capitalized here. It's not the beginning of a sentence nor a proper noun.